pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: async
-version: 2.0.5
+version: 2.0.6
```

Comment:
Was this available in -dev.23?

Do we need to bump the SDK constraint?

Reply:
Yes, these APIs were available in dev.23, and all tests for this package pass with dev.23.